coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -85,12 +86,15 @@ def parse(self):
                 ' has not been implemented yet'.format(self))
 
     def _parse_documentation_with_symbols(self, param_identifiers,
+                                          excp_identifiers,
                                           return_identifiers):
         """
         Parses documentation based on parameter and return symbols.
 
         :param param_identifiers:
             A tuple of two strings with which a parameter starts and ends.
+        :param excp_identifiers:
+            A tuple of two strings with which a exception starts and ends.
         :param return_identifiers:
             The string with which a return description starts.
         :return:
```

Comment:
The return description needs updating, since `ExceptionValue` can now also be in the return value.